Formula/go.rb

```diff
@@ -55,14 +55,18 @@ def install
 
     cd "src" do
       ENV["GOROOT_FINAL"] = libexec
+      # Unset these so that installing cgo can use its defaults
+      # instead of whatever might have been set outside of this install process
+      ENV["CC"] = nil
+      ENV["CXX"] = nil
```

Comment:
Shouldn't it depend on the compiler it was build with?

Reply:
no, it depends on what the end user has installed. I'm not sure what changed here that broke it for me, but it happened when go 1.20 was pushed to brew and this fixes it. technically i supposed gcc could be an optional dependency and these fields could match that if it's present, and disable cgo if it isn't